client/src/components/ExerciseList/ExerciseListContainer.jsx

```diff
@@ -33,3 +33,19 @@ class ExerciseListContainer extends React.Component {
 }
 
 export default ExerciseListContainer;
+
+import { connect } from 'react-redux';
+import ExerciseEventList from './ExerciseEventList';
+
+const mapStateToProps = state => {
+  const user_id = state;
+
+    // Get the exercise event list from the db
+    const user = axios.get(`/api/${user_id}`);
```

Comment:
you shouldn't make network calls here, in general you should create an action creator in `src/actions`, map it here in mapDispatchToProps, and call it on `componentDidMount`. BTW we have exerciseEvents already in the store

Reply:
Oh this answers my previous question